wazo_webhookd/plugins/subscription/service.py

```diff
@@ -22,11 +22,15 @@
 
 class SubscriptionService(object):
 
+    # NOTE(sileht): We use a class variable to ensure all instances
+    # SubscriptionService will share the same pubsub object.
```

Comment:
I don't really see the utility of the NOTE

Reply:
It's just a habit like of other code tags, the idea is just to put name of who does the comment.
So later people can ask the good person if it want more details.

Other tags:

```
FIXME(sileht):
TODO(sileht):
```

I can remove it, if needed.

Reply:
Sorry, I wanted to say, I don't really see the utility of the entire description. The code already reflect the behavior that the pubsub will be shared through all instance

Reply:
Oh, yeah I have reworked it to explain why and not what.